yarn.lock

```diff
@@ -37,6 +37,10 @@
     ansi-styles "^2.2.1"
```

Comment:
Will these changes necessitate a rerun of Yarn on our end?

Reply:
I don't think so. But it might be a good idea after every sprint to resync with Master and run yarn, just to be safe.